crates/spirv-attrib/src/lib.rs

```diff
@@ -0,0 +1,16 @@
+#![cfg_attr(target_arch = "spirv", feature(register_attr), register_attr(spirv))]
+
+use proc_macro::TokenStream;
+use std::str::FromStr;
+
+#[cfg(not(target_arch = "spirv"))]
```

Comment:
This won't do anything, I don't think, Cargo will only ever compile this for the host.

Reply:
I think it did compile it unless I put the crate behind a [target.dependency] thing

Reply:
Yep your right doesn't get compiled.